Content/AllowSpoolerPrintToFile.md

```diff
@@ -0,0 +1,10 @@
+# Allow Spooler Print To File
+
+`AllowSpoolerPrintToFile` is a sandbox setting that provides nuanced control over how sandboxed applications interact with the print spooler service.
+
+```ini
+[DefaultBox]
+AllowSpoolerPrintToFile=n
+```
```

Comment:
I think it is better to propose the new layout for all documentation settings at once in a separate pull request.

For now, please use the current layout:

https://github.com/sandboxie-plus/sandboxie-docs/blob/225d7162549d37a84b060001e9702e5ccff1a550/Content/AutoDelete.md?plain=1#L4-L10